Assets/Scripts/GameController.cs

```diff
@@ -57,7 +55,7 @@ public class GameController : MonoBehaviour {
 		// Set startup variables
 		TestStartup();
 		// Prereq server variables and functions
-		TestServer();
+		//TestServer();
```

Comment:
That was test code that should just be removed at this point since server connection is handled in StartupController now. Will remove